x-pack/plugins/security_solution/server/lib/detection_engine/signals/get_filter.ts

```diff
@@ -96,6 +97,9 @@ export const getFilter = async ({
     case 'query': {
       return queryFilter();
     }
+    case 'threat_match': {
```

Comment:
Can `case threat_match` be added to the first statement on line 93?

Reply:
Yes! That is a better spot for it.